cmd/apiserver-proxy/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"crypto/tls"
+	goflag "flag"
+	"fmt"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	utilflag "k8s.io/component-base/cli/flag"
+	"k8s.io/component-base/logs"
+	"k8s.io/klog/v2"
+	"open-cluster-management.io/cluster-proxy-addon/pkg/config"
+)
+
+const (
+	KUBE_APISERVER_ADDRESS = "https://kubernetes.default.svc"
+)
+
+func proxyHandler(wr http.ResponseWriter, req *http.Request) {
+	apiserverURL, err := url.Parse(KUBE_APISERVER_ADDRESS)
+	if err != nil {
+		klog.Errorf("KUBE_APISERVER_ADDRESS parse error: %s", err.Error())
+		return
+	}
+
+	klog.V(4).InfoS("requestURL", req.RequestURI)
+
+	if klog.V(4).Enabled() {
```

Comment:
Why not use `V(4)` directly

Reply:
In this way, the verify only do once. But since headers wouldn't be a lot, so we can prefer readability.
Fixed